src/labeler.ts

```diff
@@ -96,10 +97,21 @@ async function getLabelGlobs(
   client: ClientType,
   configurationPath: string
 ): Promise<Map<string, StringOrMatchConfig[]>> {
-  const configurationContent: string = await fetchContent(
-    client,
-    configurationPath
-  );
+  let configurationContent: string;
+
+  try {
+    configurationContent = (
+      await readFile(configurationPath, {
+        encoding: 'utf8'
+      })
+    ).toString();
+  } catch (error) {
+    core.warning(
```

Comment:
I don't think this should be a warning.

Reply:
Thank you for your comment on this, @Youssef1313. We discussed this in our team and came up to the conclusion, that making this message a warning makes output too noisy. Instead the following could de a more suitable solution
`const warningPrefix = '[warning]';
core.info(`${warningPrefix} the rest of the message`);`
@lrstanley, what do you think?

Reply:
I do agree that showing warnings for this can be a bit noisy. Decided on the approach you've mentioned. Not sure if `warningPrefix` should be defined outside of `labeler.ts`, but that's where I've stuck it for now.